iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -153,18 +172,21 @@ inline void storable_function<Capacity, signature<ReturnType, Args...>>::storeFu
     auto ptr = m_storage.template allocate<StoredType>();
     cxx::Expects(ptr != nullptr);
 
+    // AXIVION Next Construct AutosarC++19_03-A18.5.10: false positive, m_storage.allocate<T>
+    // provides a properly aligned ptr and sufficient memory (static_storage satisfies this)
     /// @NOLINTJUSTIFICATION ownership is encapsulated in storable_function and will be released in destroy
     /// @NOLINTNEXTLINE(cppcoreguidelines-owning-memory)
     ptr = new (ptr) StoredType(functor);
 
     // erase the functor type and store as reference to the call in storage
     m_callable = ptr;
-    m_invoker = invoke<StoredType>;
-    m_operations.copyFunction = copy<StoredType>;
-    m_operations.moveFunction = move<StoredType>;
-    m_operations.destroyFunction = destroy<StoredType>;
+    m_invoker = &invoke<StoredType>;
+    m_operations.copyFunction = &copy<StoredType>;
+    m_operations.moveFunction = &move<StoredType>;
+    m_operations.destroyFunction = &destroy<StoredType>;
 }
 
+// AXIVION Next Construct AutosarC++19_03-A8.4.8: output parameter required by design and for efficiency
```

Comment:
Not a good rule. Disabling candidate.

Output parameters have their place, and here it is more that the intention is to modify one or both input objects which previously exist and hence should not be constructed and returned by the function. This is due to design and efficiency reasons.